### ISSUE TITLE
Elastic transform gives identical deformation multiple times

### ISSUE THREAD

**Issue by wdeback** [stardist/augmend#8]
When using the `Elastic` transform, I would expect every call to produce another elastic deformation, as is the case with the other geometric augmentation transforms. 

However, `Elastic` seems the produce the same deformation on every call, as shown in the minimal example below: 

```python
import numpy as np 
from augmend import Elastic 
import matplotlib.pyplot as plt

x = np.zeros((100,100,3), np.float32)
x[10:50,30:40] = 1

t = Elastic(grid=5, amount=30, use_gpu=False, axis=(0,1), workers=1) 
 
fig, ax = plt.subplots(1,4)
ax[0].imshow(x)
ax[0].set_title("Original")
for i in range(3):
    ax[i+1].imshow(t(x))
    ax[i+1].set_title(f"Transform {i}")
for a in ax:
    a.axis("off")
```

![image](https://github.com/stardist/augmend/assets/22352183/5e504567-231f-4880-b343-a8b4123c0280)

**Comment by maweigert** [comment]
Ok, should be fixed in 70896a7f3

Note that `augmend` no lives on pypi, so can be installed via `pip install augmend`

**Comment by wdeback** [comment]
I can confirm the fix:

![image](https://github.com/stardist/augmend/assets/22352183/4c3ff6a4-660d-47f5-ab74-b63c00265018)
